contrib/lua/src/luaconf.h

```diff
@@ -782,9 +782,10 @@
 ** without modifying the main part of the file.
 */
 
-
-
-
+#ifdef LUA_IDSIZE
+#undef LUA_IDSIZE
+#endif
+#define LUA_IDSIZE 120
```

Comment:
Would it be possible to add this to build arguments, rather than modifying source? (Or is there a more compelling reason to modify source instead of doing this)